### PR TITLE
Feature - Add srp command

### PR DESCRIPTION
**Pull request by kshannoninnes**
Argument validation is very barebones and can be improved on a lot.

### CODE REVIEW

File: Corp/controller.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import requests
+from Corp._config import SRP_URL, SRP_URL_WITH_COMMENT, LOSS_FORMAT
```
Comment (lukasni) [inline Corp/controller.py]: Should add example `SRP_URL_WITH_COMMENT` to `_config.dist.py`

File: Corp/_config.dist.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# USAGE STRINGS
+SRP_USAGE = 'The commas are important!\nUsage: *!srp [exact pilot name to receive srp], [zkill lossmail url], ' \
```
Comment (lukasni) [inline Corp/_config.dist.py]: This file should be split out into a private `_config.py` for `SRP_URL` and `SRP_URL_WITH_COMMENT` and a public `config.py` with the data that is save to commit publicly.

File: Corp/controller.py (hunk 1 above)
Comment (lukasni) [reply Corp/controller.py]: `LOSS_FORMAT` also isn't defined, is that the same as `ZKILL_FORMAT`?

File: Corp/_config.dist.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+            '[other comments]*'
+
+# GLOBALS
+SRP_URL = 'SRP FORMRESPONSE URL HERE'
```
Comment (lukasni) [inline Corp/_config.dist.py]: This could use a comment on how the URL should be formatted and ideally how to get the URL from google apps.

**Review by lukasni — CHANGES_REQUESTED** [review]

**Comment by kshannoninnes** [comment]
Added user parameter to all command functions and updated the SRP command to log the submitting user.

The SRP command is the only one utilizing it right now, but it opens the door for more detailed logging and tracking command abuse.

**Review by lukasni — APPROVED** [review]

**Pull request merged**